spyderplugins/p_breakpoints/breakpoints.py

```diff
@@ -14,10 +14,10 @@
 
 # Local imports
 from spyderlib.baseconfig import get_translation
-_ = get_translation("p_breakpoints", dirname="spyderplugins")
+_ = get_translation("p_breakpoints", dirname="spyderplugins.p_breakpoints")

```

Comment:
I gues this function would be clear if we added something like a flag to tell it is a plugin... an user/developer should not need to give the directory info..., so it would be better if it was something like:

```python
_ = get_translation("p_breakpoints", plugin=True)
```